make/autoconf/configure

```diff
@@ -287,23 +287,28 @@ done
 # Save the quoted command line
 CONFIGURE_COMMAND_LINE="${conf_quoted_arguments[@]}"
 
-if test "x$conf_legacy_crosscompile" != "x"; then
+if test "x$conf_unsafe_crosscompile" != "x"; then
   if test "x$conf_openjdk_target" != "x"; then
-    echo "Error: Specifying --openjdk-target together with autoconf"
-    echo "legacy cross-compilation flags is not supported."
-    echo "You specified: --openjdk-target=$conf_openjdk_target and $conf_legacy_crosscompile."
-    echo "The recommended use is just --openjdk-target."
+    echo "Error: --openjdk-target was specified together with"
+    echo "incompatible autoconf cross-compilation flags."
+    echo "You specified: --openjdk-target=$conf_openjdk_target and $conf_unsafe_crosscompile."
+    echo "It is recommended that you only use --openjdk-target."
     exit 1
   else
-    echo "Warning: You are using legacy autoconf cross-compilation flags."
-    echo "It is recommended that you use --openjdk-target instead."
+    echo "Warning: You are using unsafe autoconf cross-compilation flags."
```

Comment:
Will do, thanks for the review

Reply:
Would it be better if I changed that to say the flags are misleading, or if I reverted it back to legacy in that case? (Since that seems to be the primary concern with using them)

Reply:
As the build documentation says, the standard autoconf tripled was created for building Canadian cross compilers (i.e. cross-compiling a cross-compiler). This means that they use "host" for what we call "target", and they use "target" for something that is not relevant for us. This caused a lot of confusion early on when OpenJDK started using autoconf.

If you know what you are doing, there's nothing inherently *wrong* with using the autoconf triplet. It's just confusing, due to the name clash of "target".

If you think this is not clear enough from the documenation and need to add something to the warnings printed by the configure wrapper, sure, go ahead and add a line about how this can be confusing. 

But it is not unsafe.

Reply:
Alright, will change accordingly, thanks for the review